utils/progressbar/progressbarmng.go

```diff
@@ -194,14 +205,53 @@ func (bm *ProgressBarMng) NewStringProgressBar(headline string, updateFn func()
 		mpb.BarRemoveOnComplete(),
 		mpb.PrependDecorators(
 			decor.Name(headline),
-			decor.Any(func(statistics decor.Statistics) string {
-				return updateFn()
-			}),
 		),
 	)
 	return pb
 }
 
+// A progress bar with two counters values shown on the right side of the progress bar, the first value controls what the bar shows
+// The total tasks1 can be passes by an int or *int, if you want to use it with int send nil to the optional total and done tasks1 and the wanted totalTasks to total tasks1
+// else if you want to use
```

Comment:
Let's rephrase and use proper punctionation for this description, to make it easier to read and understand.